### ISSUE TITLE
resize and reposition the "registered sign" (®) to match the size and position of the "copyright sign" (©)

### ISSUE THREAD

**Issue by robohack** [eigilnikolajsen/commit-mono#89]
I'm not sure if the current form was intended or not, but I think the "registered sign" (®) character is too small (and too much pretending to be a superscript character) and it would be better if it matched the size (and positioning) of the "copyright sign" (©) character.  It is not, so far as I know, intended to be used in superscript position (like ™ or ℠ are clearly meant to be positioned).

This discussion is a little off-topic, but contains some related information:

https://blog.teknkl.com/no-guaranteed-superscript-r-in-unicode/

**Comment by eigilnikolajsen** [comment]
Yes, thank you for bringing it up. It's really a matter of taste at this point - there's no standardized way of doing it. If more people agree to this, I will change it to one of the same size as the copyright symbol.

**Comment by robohack** [comment]
Thanks!  As-is they sure look funny side-by-side!

**Comment by eigilnikolajsen** [comment]
Agreed, but they're never used like that either!

**Comment by robohack** [comment]
Well since I argued that the "registered sign" (®) character _is not, so far as I know, intended to be used in superscript position_, I don't think I can really agree that it's not meant to be read when reading it in a phrase.  🙃

Typically it does follow a word whereas the copyright sign often seems to appear in a leading position.

On the other hand if I was reading out loud a phrase that included the "copyright sign", I probably wouldn't actually speak it either, especially since it is, at least in my experience, commonly used in conjunction with the actual word "copyright" in such phrases, so I wouldn't want to speak it twice.

In any case I do fully agree it's "a matter of taste" either way, but isn't that the case for so many aspects of font design?

For what it's worth dictionary.com shows them in the opening image to the following article, all together as I would like to see them sized and positioned:

https://www.dictionary.com/e/trademark-copyright-registered-symbols/

![Image](https://github.com/user-attachments/assets/96f1aa14-220d-4e1d-b48a-06331ab1cd24)

But it then goes on to describe it as "_the ® superscript symbol_", so how's that for consistency?  (

**Comment by kenmcd** [comment]
Usage is most often as a superscript ®.
So most fonts include it as a superscript.
But, many fonts will also include an alternate which is more appropriate for inline use (like the copyright symbol).
Keeps everybody happy.
